iterator/git.go

```diff
@@ -61,6 +61,9 @@ const (
 var (
 	gitMapMutex *sync.Mutex
 	gitMutexes  map[string]*sync.Mutex
+	// Perfect separator is a new line character since Urls, Hash, Rev or Ref
```

Comment:
Rephrase grammar please, and remember docstrings (although usually this applies to public) should start with the item name... So IOW:

> separator is a ... and is used to....

Reply:
Fixed